.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
```

Comment:
Similar to the operator workflows, can this be shared across or repositories?

Reply:
It can be added automatically if defined in the .github repository, but unfortunately there's no way to add it only to a subset of projects. This is charm specific, so we'll need to find an alternative approach.